looprpc/client.proto

```diff
@@ -664,6 +664,22 @@ message OutQuoteResponse {
     On-chain cltv expiry delta
     */
     int32 cltv_delta = 5;
+
+    /*
+    The amount to be sent off-chain.
+    */
+    int64 amount_send = 6;
```

Comment:
I think that we can leave these values out of the proto, since they're calculated from existing values and just include the `conf_target` field?

Reply:
Agreed. Got confused about the request, I'm making new changes to reflect our discussion.